gmso/core/angle_type.py

```diff
@@ -69,6 +69,11 @@ def __init__(
     def member_types(self):
         return self.__dict__.get("member_types_")
 
+    def __repr__(self):
```

Comment:
this mainly because both atom type and connection types (bond,  angle, etc) both inherit from ParametricPotential,  but atom type doesn't have `member_types`.

Reply:
But you could override just the AtomType's `__repr__` method right?

Reply:
yeah, I guess that would work too.

Reply:
but `member_types` is not an attribute of `ParametricPotential` either, so it don't fit its `__repr__`.